src/app/store/utils/node/base.ts

```diff
@@ -267,5 +273,11 @@ export const canOpenActionForm = (
     // machines can only be in a subset of statuses.
     return [NodeStatus.READY, NodeStatus.FAILED_TESTING].includes(node.status);
   }
+
+  if (nodeIsMachine(node) && actionName === NodeActions.CHECK_POWER) {
+    // "Check power" should always be shown for machines, but this action
+    // won't show up in the node.actions list, so we need to return true here.
```

Comment:
```suggestion
    // "Check power" is always shown for machines, even though it's not listed in node.actions.
```

As a side note - shouldn't the back-end be updated to include all node action types here?

Reply:
I can ask the backend team to add this